test/untitledSqlDocumentService.test.ts

```diff
@@ -0,0 +1,128 @@
+import * as TypeMoq from 'typemoq';
```

Comment:
I see there's a good number of tests added here, but Coveralls says this change drops code coverage by 32%. That's a huge change. Can we make sure we understand what's going on with coveralls's assessment before we move forward?